indymeet/templates/wagtailusers/users/create.html

```diff
@@ -0,0 +1,11 @@
+{% extends "wagtailusers/users/create.html" %}
```

Comment:
How do I get to this form? Is it part of `wagtail_urls`? I want to link it with the front end buttons I created for it already.

Reply:
Yes, I think it's part of wagtail_urls, I followed this documentation https://docs.wagtail.org/en/stable/advanced_topics/customisation/custom_user_models.html